psi4/driver/qcdb/molecule.py

```diff
@@ -566,7 +559,7 @@ def format_molecule_for_orca(self):
         """
         options = collections.defaultdict(lambda: collections.defaultdict(dict))
         self.update_geometry()
-        factor = 1.0 if self.PYunits == 'Angstrom' else psi_bohr2angstroms
+        factor = 1.0 if self.PYunits == 'Angstrom' else qcel.constants.bohr2angstroms
```

Comment:
Is there any reason the units are case sensitive?

Reply:
The _storage_ is case sensitive, like an enum. I think the _input_ has been rendered insensitive. Ping me if you find otherwise.